src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerAddAudioMetadataIntTest.java

```diff
@@ -38,27 +46,64 @@ class AudioControllerAddAudioMetadataIntTest extends IntegrationBase {
     private MockMvc mockMvc;
     @Autowired
     private ObjectMapper objectMapper;
+    @Autowired
+    private AuthorisationStub authorisationStub;
+    @Autowired
+    private DartsDatabaseStub dartsDatabaseStub;
+    @MockBean
+    private UserIdentity mockUserIdentity;
+
+    @BeforeEach
+    void beforeEach() {
+        authorisationStub.givenTestSchema();
+
+        UserAccountEntity testUser = authorisationStub.getTestUser();
+        when(mockUserIdentity.getUserAccount()).thenReturn(testUser);
+    }
 
     @Test
     void addAudioMetadata() throws Exception {
-        dartsDatabase.createCase("SWANSEA", "case1");
-        dartsDatabase.createCase("SWANSEA", "case2");
-        dartsDatabase.createCase("SWANSEA", "case3");
 
-        AddAudioMetadataRequest addAudioMetadataRequest = createAddAudioRequest(STARTED_AT, ENDED_AT, "SWANSEA");
-        MockHttpServletRequestBuilder requestBuilder = post(ENDPOINT)
-            .header("Content-Type", "application/json")
-            .content(objectMapper.writeValueAsString(addAudioMetadataRequest));
+        UserAccountEntity testUser = authorisationStub.getSystemUser();
+        dartsDatabaseStub.getUserAccountRepository().save(testUser);
+
+        dartsDatabase.createCase("Bristol", "case1");
+        dartsDatabase.createCase("Bristol", "case2");
+        dartsDatabase.createCase("Bristol", "case3");
+        AddAudioMetadataRequest addAudioMetadataRequest = createAddAudioRequest(STARTED_AT, ENDED_AT, "Bristol");
 
-        mockMvc.perform(requestBuilder)
+        MockMultipartFile audioFile = new MockMultipartFile(
+            "file",
+            "audio.mp3",
+            "audio/mpeg",
+            "Test Document (doc)".getBytes()
+        );
+
+        MockMultipartFile metadataJson = new MockMultipartFile(
+            "metadata",
+            null,
+            "application/json",
+            objectMapper.writeValueAsString(addAudioMetadataRequest).getBytes());
+
+
+        mockMvc.perform(
+            multipart(ENDPOINT)
+                .file(audioFile)
+                .file(metadataJson))
             .andExpect(status().isOk())
             .andReturn();
 
         List<HearingEntity> allHearings = dartsDatabase.getHearingRepository().findAll();
 
-        assertEquals(3, allHearings.size());
-
+        List<HearingEntity> addAudioLinkedHearings = new ArrayList<>();
         for (HearingEntity hearing : allHearings) {
+            if (hearing.getCourtCase().getCaseNumber().contains("case")) {
+                addAudioLinkedHearings.add(hearing);
+            }
+        }
+        assertEquals(3, addAudioLinkedHearings.size());
+
+        for (HearingEntity hearing : addAudioLinkedHearings) {
```

Comment:
May be missing it but can we also assert that the mpeg contents also got to the backend?